upgrade_test.py

```diff
@@ -497,16 +502,15 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
 
         # wait for the 10m read workload to finish
         self.verify_stress_thread(read_10m_cs_thread_pool)
+        self.fill_and_verify_db_data('after upgraded two nodes')
 
-        # read workload (20m)
-        self.log.info('Starting c-s read workload for 20m')
-        stress_cmd_read_20m = self.params.get('stress_cmd_read_20m')
-        read_20m_cs_thread_pool = self.run_stress_thread(stress_cmd=stress_cmd_read_20m)
+        # read workload (80m)
```

Comment:
Why 80m?

Reply:
I explained in the commitlog: https://github.com/scylladb/scylla-cluster-tests/pull/1438/commits/0c89a8db8a4cc6804f6a96f0954e2af31d897de6